test/unit/util.test.ts

```diff
@@ -65,3 +67,45 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    let createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour.js'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
```

Comment:
Why not moving in `extensions` and testing with `wiktionary_fr.js`? Seems lighter!?

Reply:
@kelson42 this test is for case one(searching for file in current wokring directory so I guess this is the right place isn't it?)

Reply:
ok, see my previous comment.